package.json

```diff
@@ -1,92 +1,95 @@
-        "@types/jest": "26.0.22",
-        "@types/jquery": "3.5.5",
-        "@types/lodash": "4.14.168",
-        "@types/lusca": "1.6.2",
-        "@types/mysql": "^2.15.18",
-        "@types/node": "14.14.25",
-        "@types/passport": "1.0.5",
-        "@types/passport-steam": "^1.0.1",
-        "@types/request": "2.48.5",
-        "@types/request-promise": "4.1.47",
-        "@types/shelljs": "0.8.8",
-        "@types/steamid": "^1.1.0",
-        "@types/supertest": "2.0.10",
-        "@types/winston": "2.4.4",
-        "@typescript-eslint/eslint-plugin": "4.14.2",
-        "@typescript-eslint/parser": "4.14.2",
-        "chai": "4.3.0",
-        "concurrently": "6.0.2",
-        "eslint": "7.19.0",
-        "jest": "26.6.3",
-        "nodemon": "2.0.7",
-        "sass": "1.32.6",
-        "shelljs": "0.8.4",
-        "supertest": "6.1.3",
-        "ts-jest": "26.5.0",
-        "ts-node": "9.1.1",
-        "typescript": "4.1.3"
-    }
+  "name": "helix-logs",
+  "version": "1.0.0",
+  "description": "A logger for the Helix roleplaying framework written in Node.js with Typescript.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wildflowericecoffee/helix-logs"
+  },
+  "author": "wildflowericecoffee",
+  "license": "MIT",
+  "scripts": {
+    "build-sass": "sass src/public/css/main.scss dist/public/css/main.css",
+    "build-ts": "tsc",
+    "build": "npm run build-sass && npm run build-ts && npm run lint && npm run copy-static-assets",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "debug": "npm run build && npm run watch-debug",
+    "lint": "tsc --noEmit && eslint \"**/*.{js,ts}\" --quiet --fix",
+    "serve-debug": "nodemon --inspect dist/server.js -e js,ejs",
+    "serve": "node dist/server.js",
+    "start": "npm run serve",
+    "test": "jest --forceExit --coverage --verbose",
+    "watch-debug": "concurrently -k -p \"[{name}]\" -n \"Sass,TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-sass\" \"npm run watch-ts\" \"npm run serve-debug\"",
+    "watch-node": "nodemon dist/server.js -e js,ejs",
+    "watch-sass": "sass --watch src/public/css/main.scss dist/public/css/main.css",
+    "watch-test": "npm run test -- --watchAll",
+    "watch-ts": "tsc -w",
+    "watch": "concurrently -k -p \"[{name}]\" -n \"Sass,TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-sass\" \"npm run watch-ts\" \"npm run watch-node\""
+  },
+  "dependencies": {
+    "async": "3.2.4",
+    "body-parser": "1.20.1",
+    "compression": "1.7.4",
+    "cookie-session": "^2.0.0",
+    "cors": "^2.8.5",
+    "dotenv": "16.0.3",
+    "ejs": "^3.1.8",
+    "errorhandler": "1.5.1",
+    "express": "^4.18.2",
+    "express-rate-limit": "^6.7.0",
+    "express-session": "^1.17.3",
+    "fs": "0.0.1-security",
+    "https": "^1.0.0",
+    "json-beautify": "^1.1.1",
+    "lusca": "1.7.0",
+    "mysql2": "^3.1.2",
+    "passport": "github:joeyguerra/passport#missing-regenerate-on-req",
```

Comment:
You may wonder why I'm using a GitHub repo for this, but PassportJS hasn't fixed a problem upstream:
https://github.com/jaredhanson/passport/issues/907

This concerns the version 0.6.0 and fixes a vulnerability in 0.5.3, which is why using the custom version of PassportJS till the fix has been implemented upstream.